### ISSUE TITLE
Export to openrefine

### ISSUE THREAD

**Issue by Wycklife** [IDEMSInternational/R-Instat#4511]
I have been trying to test this dialog (Prepare>Check Data>Export to openrefine). Whenever i try to export any data set to Openrefine,this bug pops up, what might be the problem?

![image](https://user-images.githubusercontent.com/33898404/38565515-3090faa6-3cea-11e8-8520-d341cb9a3622.png)

![image](https://user-images.githubusercontent.com/33898404/38565554-4ca37af2-3cea-11e8-9d5f-835475f3300b.png)

**Comment by dannyparsons** [comment]
Open Refine is a software and this needs to be installed on your machine to be able to use this dialog. @lilyclements is that correct?

**Comment by lilyclements** [comment]
Yes, so you can install it from here - http://openrefine.org/download.html

**Comment by dannyparsons** [comment]
If users know what OpenRefine is they should they need it installed, but this information will be important to include in the help file for the dialog.

**Comment by Wycklife** [comment]
Thanks for your feedback, but i did all that, i tried it in R and it was successful, in Rinstat,  it brings out those two bugs, it doesn't open the browser.

**Comment by lilyclements** [comment]
Interesting. In R are you running the `write.csv` function as well as the `refine_upload`?
From a quick google, I think the error is coming from the `write.csv` part of this (this is something we need in R-Instat because of the instatdataobject (I think! It's been a long time!) but is not necessary in R)

I'm happy to skype later to see it through on your laptop, perhaps around 2pm your time?
